src/client/components/app/DetailCard/index.tsx

```diff
@@ -8,10 +8,17 @@ const StyledCardElement = styled(CardElement)<{ stripes?: boolean }>`
   display: flex;
   justify-content: center;
   margin: 0;
-  padding: 0.6rem ${({ theme }) => theme.card.padding};
+  padding: calc(${({ theme }) => theme.card.padding} / 4) calc(${({ theme }) => theme.layoutPadding} / 2);
```

Comment:
It is just using layout and card padding as relative sizes to calc, it is done in other childs of the detail card, it is just so if we modify the layoutpadding or card padding it will still look good with any value

Reply:
that's good, can we add a comment for it here?

Reply:
Commented in aef61c0699b977d420319fb3f99856c4aa59fcdd